build-libs/redirects.js

```diff
@@ -309,6 +309,16 @@ async function buildDevPortalRedirects() {
 			destination: '/waypoint/integrations/hashicorp/:slug',
 			permanent: true,
 		},
+		{
+			source: '/:path/downloads',
+			destination: '/:path/install',
+			permanent: true,
```

Comment:
If we wanted to be really explicit we could [use a regex ](https://nextjs.org/docs/pages/api-reference/next-config-js/redirects#regex-path-matching)to only match on product slugs we expect.

Reply:
Sure, I can do that. Just out of curiosity, is that to prevent any unintentional redirects or is there more to it than that?

Reply:
Yep exactly just to be really explicit and prevent unintentional redirects!